deploy.sh

```diff
@@ -2,41 +2,17 @@
 
 set -euxfo pipefail;
 
-declare -r LATEST_RACKET_VERSION="7.3";
+source "_common.sh";
```

Comment:
Is `source` the standard way of importing bash "modules"? Are there other ways?

Reply:
I'm only aware of `source` and its alias`.` (as in, `. "_common.sh"`). `.` is more portable (it works in plain  `sh`), but harder to read. The command tells the bash interpreter to execute that file in the context of the current shell, so it's a bit like an `include` in other languages rather than a real module, because it's as if the commands get inlined into the current script.